deploy/bootstrap.py

```diff
@@ -1205,47 +1277,45 @@ def main():  # noqa: C901
             env_vars = ''
 
         if env_type == 'dev':
-            if args.env_name is not None:
-                prefix = f'load_{args.env_name}'
+            if conda_env_name is not None:
+                prefix = f'load_{conda_env_name}'
             else:
                 prefix = f'load_dev_polaris_{polaris_version}'
         elif env_type == 'test_release':
             prefix = f'test_polaris_{polaris_version}'
         else:
             prefix = f'load_polaris_{polaris_version}'
 
-        script_filename = write_load_polaris(
-            conda_template_path, activ_path, conda_base, env_type,
-            activ_suffix, prefix, conda_env_name, spack_script, machine,
-            env_vars, args.conda_env_only, source_path, args.without_openmp,
-            polaris_version)
+        script_filename = \
+            _write_load_polaris(options, prefix, spack_script, env_vars)
 
-        if args.check:
-            check_env(script_filename, conda_env_name, logger)
+        print(options)  # XXX Dev
+        _check_env(options, script_filename, conda_env_name)
```

Comment:
@altheaden, is it getting to this print fine and `_check_env()` is failing?  If so, it could just be broken and I just haven't run with `--check` in forever.

Reply:
Good idea, let me check on this theory.

Reply:
Sorry, just got back from having some lunch. Looks like you were right! I must have accidentally deleted the if statement above this and so it was automatically running the env check even without `--check`. I just ran again after restoring the if statement and it worked. Want me to make an issue about `check_env()` being broken?

Reply:
Found the issue and fixed in #217